README.md

```diff
@@ -39,8 +39,16 @@ ln -fs openshift-ansible-*<version>* openshift-ansible
 
 | openshift-ansible url     | OpenShift version | 
 |:-------------------------:|:-----------------:|
+| [3.5.77-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.5.77-1.tar.gz) | OCP 3.5.x |
 | [3.4.50-1](https://github.com/openshift/openshift-ansible/archive/openshift-ansible-3.4.60-1.tar.gz) | OCP 3.4 |
 
+* Run `ansible-galaxy` to pull in the necssary requirements for the CASL provisioning:
```

Comment:
@etsauer it can all be done from the control host - i.e.: no need to run it from localhost. I think we need to rework a major portion of this README file anyway as it's seems to be confusing to users, so maybe we can look at figure out how to better structure this as a follow-up PR (including how to balance information in the top level README v.s. the control host README).